libs/platform/src/lib/wizard-generator/components/wizard-generator-step/wizard-generator-step.component.ts

```diff
@@ -146,6 +171,11 @@ export class WizardGeneratorStepComponent implements OnDestroy, OnChanges {
             this.formsCreated.emit(this._forms);
         }
 
+        if (this._hiddenFormGroupValues[key]) {
+            form?.patchValue(this._hiddenFormGroupValues[key]);
+            form?.updateValueAndValidity({ emitEvent: false, onlySelf: true });
```

Comment:
`patchValue` calls `updateValueAndValidity` internally. should we call it again?